Examples/javascript/array/runme.js

```diff
@@ -0,0 +1,7 @@
+var example = require("example");
+
+var arr = [1, 2, 3, 4, 5];
+
+console.log(example.sum1(arr, arr.length));
+console.log(example.sum2(arr));
+console.log(example.data3)
```

Comment:
All other examples in javascript do that as well. It checks that things compile with node, it will fail if not compiled.